webroot/src/components/CollapseCaretButton/CollapseCaretButton.vue

```diff
@@ -12,8 +12,19 @@
         tabIndex="0"
         class="collapse-caret-button"
     >
-        <img v-if="isUp" class="caret-img" src="@assets/icons/up-caret.svg" alt="Collapse Button Logo" />
-        <img v-else class="caret-img" src="@assets/icons/down-caret.svg" alt="Expand Button Logo" />
+        <svg
```

Comment:
Not in this file, but rather in the TS file there is a missing `emits` declaration in `@Component`; resulting in the following console warning:

<img width="932" alt="Screenshot 2024-11-20 at 10 58 05 AM" src="https://github.com/user-attachments/assets/ac1b8fd0-7db6-4bfd-9da7-90e1dd70f880">